src/theory/datatypes/theory_datatypes.cpp

```diff
@@ -596,56 +597,70 @@ Node TheoryDatatypes::expandDefinition(LogicRequest &logicRequest, Node n) {
     }
   }
     break;
-  default:
-    return n;
+    case TUPLE_UPDATE:
```

Comment:
The formatting here is off: `case` shouldn't be at the same level of indentation as `break`. Maybe reformat the whole switch statement?